### ISSUE TITLE
Two quotes on a line in a verbatim block do not convert to html correctly

### ISSUE THREAD

**Issue by kensmosis** [jgm/pandoc#3862]
NOTE:  please view this bug report as text, not markdown.  When I preview this bug report on my browser, it implicitly converts to html -- so you won't see what I'm talking about!

I ran into an odd problem when using the verbatim | environment.  If I have a single quotation on the line (bounded by two backticks on one side and two apostrophes on the other) in the markdown, then it converts correctly to html.  However, if I have two such quotations (perhaps separated by some non-quoted text), then it does not.  

Here is a minimal example which seems to produce the problem (note that the end quotes are two-single apostrophes, not an ascii double-quote).   I converted it as pandoc -o foo.html foo.md   (but the problem persists if you use -S).  

Case 1 :  Problem 

| ``my first quote,'' he laughed.  ``Now here is my second.''

generates

<div style="white-space: pre-line;"><code>my first quote,'' he laughed.</code>Now here is my second.''</div>

Case 2 :  No Problem

On the other hand, 

| ``my first quote,'' he laughed.

produces the expected

<div style="white-space: pre-line;">``my first quote,'' he laughed.</div>

I'm using Pandoc Version 1.19.2.1 on Ubuntu 16.04.  

Thanks for your help with this!

**Comment by kensmosis** [comment]
Again, please make sure to view this bug report as text (i.e. look at its source).  Otherwise, you will think I just forgot the initial 2 backticks in my example, and have no idea what I am talking about!  Giving examples of bugs in markdown when the bug report itself automatically is treated as markdown is problematic!

**Comment by jgm** [comment]
| ``my first quote,'' he laughed.  ``Now here is my second.''

In Markdown, material between ``..`` is treated as code
(verbatim text, rendered in a monospace font).  That's what
pandoc is doing:

    <div style="white-space: pre-line;"><code>my first quote,'' he
    laughed.</code>Now here is my second.''</div>

I assume that's not what you want here, you mean to have
regular quotation marks.  The best way to do this is to
use unicode curly quotes in your source.  Alternatively,
you can use straight double quotes and the --smart option;
note, however, that this will only work in line blocks
when the matching quotes are on the same line.  (Actually,
that's something I think we could easily fix.)

+++ kensmosis [Aug 22 17 16:07 ]:

**Comment by kensmosis** [comment]
Thanks for the response, John!  I see what you mean; that definitely explains the weird behavior.  I think that since a lot of people are starting to use markdown for writing (especially as ebooks become more prevalent), it may not be a bad idea to do proper quote-matching.  But that's your call, of course.  FYI, I did try the smart (-S) option, but it seemed to make no difference in the example I gave (where the quotes are on the same line).
